apache/superset
# feat(dashboard): Let users re-arrange native filters

### SUMMARY
Allows users to re-arrange the dashboard native filters

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
![ezgif com-gif-maker](https://user-images.githubusercontent.com/12539911/128769734-46e4d524-a0f6-4759-97c8-d4db653853b3.gif)

### TESTING INSTRUCTIONS
 - Click Edit icon (✏️) in native filters section on the left side of the dashboard
 - Re-arrange the native filters in the opened modal box
 - Save and verify that the order has changed in the UI

### ADDITIONAL INFORMATION
